vue-handler.js

```diff
@@ -24,7 +24,7 @@ function getTypeInfo(type) {
     return typeToInfo[type3] || typeToInfo['unk'];
 }
 function readableSize(size) {
-    const sizeUnits = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+    const sizeUnits = ['බයිට', 'කි.බ.', 'මෙ.බ.', 'ගි.බ.', 'ටෙ.බ.'];

```

Comment:
I have never seen like this anywhere. To stop confusing people I would suggest to keep them unchanged.

Reply:
Agree, best to keep them unchanged.